packages/rs-dpp/src/asset_lock/mod.rs

```diff
@@ -0,0 +1,9 @@
+use crate::asset_lock::reduced_asset_lock_value::AssetLockValue;
+
+pub mod reduced_asset_lock_value;
+
+pub enum StoredAssetLockInfo {
+    Present,
+    PresentWithInfo(AssetLockValue),
```

Comment:
Please add comments to explain what these variants mean.

Reply:
Done